corehq/toggles.py

```diff
@@ -1843,14 +1843,6 @@ def _commtrackify(domain_name, toggle_is_enabled):
     relevant_environments={'icds', 'icds-staging'},
 )
 
```

Comment:
This feature flag was just temporarily put not-in-use https://github.com/dimagi/commcare-hq/pull/27588/files and would be needed once the feature is rolled out. Feel free to retain it or remove it, I can just add it back if needed.

Reply:
OK, I'll add it back